assembl/alembic/versions/e757aefa55e1_convert_thematics_medias_in_attachments.py

```diff
@@ -0,0 +1,70 @@
+"""convert_thematics_medias_in_attachments
```

Comment:
Do you know you can create the step with `alembic -c local.ini revision -m"Convert thematics medias in attachments"`? The generated file will be the same, but you will have a proper string here.

Reply:
good to know!